amnestywebsite/humanity-theme
# Changes ES translation for Most Relevant filter

Epic: https://github.com/amnestywebsite/humanity-theme/issues/417 
Issue: https://github.com/amnestywebsite/amnesty-wp-theme/issues/2974

Changes ES translation for Most Relevant filter in dropdown

**Steps to test**:
1. Navigate to /es/search
2. Use the filter dropdown and check Lo m ás relevante has now been changed to Más relevante (Most Relevant)

**Considerations**:
- Localisation

Example: http://bigbite.im/i/spXOl7
